spec/integration/override_create_spec.rb

```diff
@@ -0,0 +1,38 @@
+RSpec.describe ROM::Repository, '.command' do
+  it 'should use customer create command' do
+    config = ROM::Configuration.new(:memory)
+
+    config.relation(:accounts) do
+      register_as :accounts   
+
+      # avoid pulling in plugin, use :key_inference
+      def base_name
+        name
+      end
+    end
+
+    class CreateAccount < ROM::Commands::Create[:memory]
```

Comment:
If I could workout how to do this without creating a stand alone constant I could bring this code more inline with the existing tests. @solnic?